norms.md

```diff
@@ -0,0 +1,22 @@
+##Fishtown Analytics Norms##
```

Comment:
Maybe the difference is meaningless. But I generally think of it like this:

Values describe what you aspire to be
Norms describe how you act so you can achieve these values

Right now, the values and norms described (as I've divided them out) may not perfectly adhere to the above. And maybe it isn't a distinction worth making, anyway. You're right we should probably discuss a bit more in person.